src/components/link.js

```diff
@@ -21,7 +21,11 @@ export default {
     exact: Boolean,
     append: Boolean,
     replace: Boolean,
-    activeClass: String
+    activeClass: String,
+    event: {
+      type: String,
```

Comment:
I think it should also accept `Array` type, e.g. `<router-link :event="['mousedown', 'touchstart']"></router-link>`